apple/MarkdownTextInputDecoratorComponentView.mm

```diff
@@ -46,20 +47,35 @@ - (instancetype)initWithFrame:(CGRect)frame
   if (self = [super initWithFrame:frame]) {
     static const auto defaultProps = std::make_shared<const MarkdownTextInputDecoratorViewProps>();
     _props = defaultProps;
+    _observersAdded = false;
+    _markdownUtils = [[RCTMarkdownUtils alloc] init];
```

Comment:
Was there a reason this was originally recreated on every mount, or is creating it once safe?

Reply:
I just wanted to be extra safe and create a new instance of `RCTMarkdownUtils` each time the component is mounted to avoid using previous styles or parser in case something goes wrong. We can keep it your way if you want to.